models/fv_aepg_lag_pol_att.go

```diff
@@ -0,0 +1,95 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/container"
+)
+
+const (
+	DnfvAEPgLagPolAtt        = "uni/tn-%s/ap-%s/epg-%s/rsdomAtt-[%s]/epglagpolatt"
+	RnfvAEPgLagPolAtt        = "epglagpolatt"
+	ParentDnfvAEPgLagPolAtt  = "uni/tn-%s/ap-%s/epg-%s/rsdomAtt-[%s]"
+	FvAEPgLagPolAttClassName = "fvAEPgLagPolAtt"
+)
+
+type ApplicationEPGLagPolicy struct {
+	BaseAttributes
+	NameAliasAttribute
```

Comment:
It does not seems that this class has NameAlias. Please check and cleanup if needed. (It also does not have description)

Reply:
ok